staff/johana-ludwig/cv/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Johana Ludwig</title>
+</head>
+<body>
+    <header>
+        <h1>Johana Ludwig</h1>
+        <h2>Online Marketing y Frontend Development</h2>
+        <img src="" alt="Foto de perfil Johana Ludwig">
+    </header>
+
+    <main>
+    <section>
+        <table>
+            <tr>
+                <td></td>
+                <td>
+                    <h3>Hola, soy Johana Ludwig</h3>
+                        <ul>
+                            <li>Campañas de marketing online</li>
+                            <li>SEO y SEA</li>
+                            <li>Apasionada al desarrollo web</li>
+                        </ul>
+                </td>
+            </tr>               
+        </table>
+    </section>
+
+    <section>
+        <h2>Experiencia laboral</h2>
```

Comment:
You have to use a `<header>` inside the section. Maybe in your example you can put 
```html
<section>
<header><h3>Hola, soy Johana Ludwig</h3></header>
</section>
```